owncloudApp/src/main/java/com/owncloud/android/presentation/sharing/shares/PublicShareDialogFragment.kt

```diff
@@ -364,11 +450,14 @@ class PublicShareDialogFragment : DialogFragment() {
      */
     private fun onPasswordFocusChanged(hasFocus: Boolean) {
         if (hasFocus) {
+            binding.saveButton.isEnabled = false
```

Comment:
Why disabling it when it has focus and the opposite?

Reply:
I put this logic in for a minimal improvement in the UI. If we disable the save button when the input password has focus, in the first instance, it will not allow saving if you do not write anything. We assume that the user enables the password because he wants to set a password.
If we do not disable the button, the behavior would be the following:
  - Ocis: The save button would appear enabled and if you click on it it would give you a server error: `It was not possible to share this file or folder because unknown error`. The message does not give enough information to know what is happening.
- OC10: The save button would appear enabled and there would be no error when saving the password.

When the password input does not have focus, it does not provide functionality, because it is managed in other parts of the code. So I remove it.